src/confluent_kafka/avro/cached_schema_registry_client.py

```diff
@@ -158,6 +189,13 @@ def _send_request(self, url, method='GET', body=None, headers={}):
         if method not in VALID_METHODS:
             raise ClientError("Method {} is invalid; valid methods include {}".format(method, VALID_METHODS))
 
+        if url.startswith('https') and self.passwordprotected:
```

Comment:
There is nothing like `passwordprotected` anymore? Please configure your IDE to identify these kind of problems.